Sources/Nimble/Matchers/Contain.swift

```diff
@@ -58,13 +58,13 @@ public func contain(_ items: Any?...) -> Predicate<NMBContainer> {
 }
 
 public func contain(_ items: [Any?]) -> Predicate<NMBContainer> {
-    return Predicate.fromDeprecatedClosure { actualExpression, failureMessage in
-        failureMessage.postfixMessage = "contain <\(arrayAsString(items))>"
-        guard let actual = try actualExpression.evaluate() else { return false }
-        return items.all { item in
+    return Predicate.simple("contain <\(arrayAsString(items))>") { actualExpression in
+        guard let actual = try actualExpression.evaluate() else { return .fail }
+        let matches = items.all { item in
             return item != nil && actual.contains(item!)
```

Comment:
Thoughts on restructuring this to avoid the force-unwrap?

Reply:
Addressed in 9bbe098.